cc.cubocore.CoreTime.appdata.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 CuboCore -->
+
+<component type="desktop-application">
+    <provides>
+        <id>cc.cubocore.CoreTime</id>
+    </provides>
+    <replaces>
+        <id>org.cubocore.CoreTime</id>
+    </replaces>
```

Comment:
it's either provides or replace not both. Also it's missing the actual id.

Reply:
hi, as the appdata is removed, pls check if this is correct or not https://gitlab.com/cubocore/coreapps/coretime/-/blob/master/cc.cubocore.CoreTime.metainfo.xml?ref_type=heads#L6

Reply:
> hi, as the appdata is removed, pls check if this is correct or not https://gitlab.com/cubocore/coreapps/coretime/-/blob/master/cc.cubocore.CoreTime.metainfo.xml?ref_type=heads#L6

Add:

```
    <provides>
        <id>org.cubocore.CoreTime</id>
    </provides>
```

too